src/job.ts

```diff
@@ -36,14 +37,6 @@ export class JobSequenceMsgBuilder {
   }
 }
 
-export const base64encode = (input: object): string => {
```

Comment:
move to `utils/contract.ts` for now, we put all helpers that want to be exported there for now